.github/workflows/ci-dgraph-tests.yml

```diff
@@ -58,6 +58,10 @@ jobs:
           # build the test binary
           cd t; go build .
           # run the tests
-          ./t --skip systest/backup,systest/online-restore,systest/loader,tlstest
+          ./t --coverage --skip systest/backup,systest/online-restore,systest/loader,tlstest
           # clean up docker containers after test execution
           ./t -r
+      - name: Install goveralls
+        run: go install github.com/mattn/goveralls@latest
+      - name: Send coverage
+        run: cd t && goveralls -repotoken ${{ secrets.COVERALLSIO_TOKEN }} -coverprofile=coverage.out
```

Comment:
we should use this https://github.com/marketplace/actions/coveralls-github-action

I will try to make the change on top of your PR for this

Reply:
The coveralls seems to not have cover profile option for some reason.